docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -48,6 +28,7 @@ To get started, you will need to implement a [Notification Service Extension](ht
 * Add a product name (use `NotifeeNotificationService` to follow along) and click Finish
```

Comment:
This is not directly related to this PR but has come up recently with two people and I want to take the chance to help future people

```suggestion
* Add a product name (use `NotifeeNotificationService` to follow along) and click Finish
  * You may use Swift or Objective-C where it says 'language' but be sure to follow the correct section below depending on your implementation language selection
```